README.md

```diff
@@ -89,61 +89,143 @@ builds like `gcc` generate a *lot* of output), and builds timing out
 
 ## Testing conda builds locally
 
-Given a fairly empty *disposable* Ubuntu x86-64 test environment (eg,
-created with Docker, or Vagrant), it is possible to simulate *part* of
-what Travis CI will do to test building individual toolchain architectures
-locally.
+It is recommend to build these packages in a fresh disposable environment
+such as a clean docker container.
+While the goal is for the conda environment to be totally self contained,
+there is a constant battle to make sure this happens.
 
-This can be done with something like:
+The commands from the following subsections are enough to build a package
+in a clean docker container based on the *ubuntu* Docker image.
 
-```
-sudo apt-get update
-sudo apt-get install wget git
+### Prerequisites
+
+The only required prerequisites are:
+* [Git](https://git-scm.com/),
+* Conda installed and initialized, e.g., using
+[Miniconda](https://docs.conda.io/en/latest/miniconda.html)
+(which includes self contained versions of the required *python3* with
+*pip* and *setuptools* tooling),
+* [conda-build-prepare](https://github.com/litex-hub/conda-build-prepare)
+and of course this repository cloned to a local directory.
```

Comment:
I would try avoiding "*of course*" and such. I also use them more than I should, but they don't help much.

I suggest:

"*Apart from cloning this repository to a local directory, the following prerequisites are required:*"

Reply:
You're right, it's much better.